pymc3/distributions/continuous.py

```diff
@@ -1654,6 +1655,93 @@ def logcdf(self, value):
         )
 
 
+class AsymmetricLaplace(Continuous):
+    r"""
+    Asymmetric-Laplace log-likelihood.
+
+    The pdf of this distribution is
+
+    ..math::
+        {f(x|\\b,\kappa) =
+            \left({\frac{\\b}{\kappa + 1/\kappa}}\right)\,e^{-(x)\\b\,s\kappa ^{s}}}
+        where s = sgn(x)
```

Comment:
No, we are not using `m` here. we have considered `m` to be 0.  I am following [`this`](https://github.com/epidemics/COVIDNPIs/blob/manuscript/epimodel/pymc3_distributions/asymmetric_laplace.py) codebase as I was told to.
In case, If something is to be added, do let me know.

Reply:
That's definitely a good parameter to add. It is even there in PyMC3's [Laplace](https://docs.pymc.io/api/distributions/continuous.html#pymc3.distributions.continuous.Laplace) distribution. In order to be consistent with the given implementation, we can initialize `mu` with 0.

Reply:
could you please tell me where to initialise `mu` as 0 and does it need to be a tensor variable ?

Reply:
Yes. You can use it as a default argument in `init` method. And then wrap it in `as_tensor_variable`